static/src/javascripts/projects/common/utils/url.js

```diff
@@ -63,6 +63,12 @@ define([
                     }).join('&');
             },
 
+            getPath: function (url) {
+                var a = document.createElement('a');
+                a.href = url;
+                return a.pathname;
+            },
```

Comment:
:eyes: Why not just use a RegEx rather than creating an element in memory:
```js
  var matches = url.match(/^((http[s]):\/)?\/?([^:\/\s]+)((\/\w+)*\/)([\w\-\.]+[^#?\s]+)(.*)?(#[\w\-]+)?$/g;);
  return (matches && matches[5]) ? matches[5] : undefined;
```